UnitTests/Catalog/Services/CommentsRepository/GetAllCommentsTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Target = Catalog.Services.Comments.CommentsRepository;
+using Common.Interfaces;
+using Common.Models.Comments;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace UnitTests.Catalog.Services.CommentsRepository
+{
+    [TestClass]
+    public class GetAllCommentsTests
```

Comment:
Not all the test cases are covered for this method.
I understand that we will work with unit tests extensively later, covering our logic more thoroughly. But this is a bad practice to leave unit tests for the uncertain future.